cluster/test-network.sh

```diff
@@ -24,7 +24,7 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
-KUBE_CONFIG_FILE="config-default.sh"
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+export KUBE_CONFIG_FILE="config-default.sh"
 
-${KUBE_ROOT}/hack/ginkgo-e2e.sh --ginkgo.focus=Networking
+"${KUBE_ROOT}/hack/ginkgo-e2e.sh" --ginkgo.focus=Networking
```

Comment:
```
In ./cluster/test-network.sh line 30:
${KUBE_ROOT}/hack/ginkgo-e2e.sh --ginkgo.focus=Networking
^-- SC2086: Double quote to prevent globbing and word splitting.
```